docs/Technical-Documentation/Architecture-Decision-Record/008-deployment-flow.md

```diff
@@ -19,18 +19,21 @@ Additionally, as of January 2021, the project has only a single deployment envir
 Deploy Environment | Cloud.gov Space | Cloud.gov Dev Access | Role                                             | Deploys when ...                                  |
 -------------------|-----------------|----------------------|--------------------------------------------------|---------------------------------------------------|
 Dev                | Tanf-Dev        | Vendor & Gov      | Deploy code submitted for gov review             | Relevant github label assigned as shown below     |
-Staging            | Tanf-Staging    | Gov               | Deploy code once gov-approved                    | Code merged to `raft-tech/TANF-app:raft-tdp-main` |
-Production         | Tanf-Prod       | Gov                  | Deploy code tested in staging & ready for prod   | Code merged to `HHS/TANF-app:main`                |  
+Staging            | Tanf-Staging    | Gov               | Deploy code once gov-approved                    | Code merged to `raft-tech/TANF-app:develop` |
```

Comment:
noting for future reference that we need to update this when we have setup a flow for `hhs:main` to deploy to staging. then i suppose we'll have vendor staging again.